mining/worker.go

```diff
@@ -47,6 +47,11 @@ type Worker interface {
 	Mine(runCtx context.Context, base types.TipSet, nullBlkCount int, outCh chan<- Output) bool
 }
 
+// WorkerSigner is the interface used for signing blocks and tickets.
+type WorkerSigner interface {
+	CreateTicket(proof proofs.PoStProof, signerPubKey []byte) (types.Signature, error)
```

Comment:
This should probably be SignBytes and FindAddressFromPublicKey or just SignBytesByPublicKey from wallet. See comment in wallet code.

Reply:
Also the interface should be `TicketSigner` since that's what we're using it for.